### PR TITLE
Slider fix

### PR DESCRIPTION
**Pull request by determin1st**
this fixes #3762 #15011 #9553 #15083

codepen: http://codepen.io/determin1st/pen/RVxKEJ/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ dist
 bower_components
 node_modules
 .sizecache.json
+*.swp
```
Comment (scottgonzalez) [inline .gitignore]: Please don't modify this file. You should add this to your own [`core.excludesfile`](https://help.github.com/articles/ignoring-files/).

File: ui/widgets/slider.js [hunk 2]
```diff
@@ -69,19 +69,30 @@ return $.widget( "ui.slider", $.ui.mouse, {
 	numPages: 5,
 
 	_create: function() {
+        var a, b;
```
Comment (scottgonzalez) [inline ui/widgets/slider.js]: These are not acceptable variable names.

File: ui/widgets/slider.js [hunk 3]
```diff
@@ -69,19 +69,30 @@ return $.widget( "ui.slider", $.ui.mouse, {
...
+        ////
```
Comment (scottgonzalez) [inline ui/widgets/slider.js]: Why is this here?

File: ui/widgets/slider.js [hunk 4]
```diff
@@ -69,19 +69,30 @@ return $.widget( "ui.slider", $.ui.mouse, {
...
 		this._detectOrientation();
 		this._mouseInit();
 		this._calculateNewMax();
-
 		this._addClass( "ui-slider ui-slider-" + this.orientation,
 			"ui-widget ui-widget-content" );
-
+        ////
+        // refine values
+        // default
```
Comment (scottgonzalez) [inline ui/widgets/slider.js]: Please follow the existing code style.

File: ui/widgets/slider.js [hunk 5]
```diff
@@ -178,63 +189,76 @@ return $.widget( "ui.slider", $.ui.mouse, {
...
-				( parseInt( closestHandle.css( "marginTop" ), 10 ) || 0 )
-		};
-
-		if ( !this.handles.hasClass( "ui-state-hover" ) ) {
-			this._slide( event, index, normValue );
-		}
-		this._animateOff = true;
-		return true;
-	},
+    _mouseCapture: function(event) { // {{{
```
Comment (scottgonzalez) [inline ui/widgets/slider.js]: Why did you change the indentation? Why are there all of these weird comments?

**Comment by scottgonzalez** [comment]
Without a useful diff, this is unlikely to actually be reviewed.

File: ui/widgets/slider.js (hunk 3 above)
Comment (determin1st) [inline ui/widgets/slider.js]: it's for visual percepcion only

**Comment by scottgonzalez** [comment]
Closing due to inactivity. Please file a new PR if you continue working on this. Thanks.
